### PR TITLE
first translations

### PR DESCRIPTION
**Pull request by alexsavio**

### CODE REVIEW

File: p3/admin.py [hunk 1]
```diff
@@ -721,9 +726,9 @@ def _calc_prices(order_id, items):
     for oid, items in grouped.items():
         _calc_prices(oid, items)
 
-    # dopo l'utilizzo di _calc_prices ottengo dei prezzi che non trovo
-    # più tra le tariffe ordinarie, raggruppo gli OrderItem risultanti
-    # per codice tariffa e nuovo prezzo
+    # after using _calc_prices abtain the prices not found anymore
```
Comment (matrixise) [inline p3/admin.py]: obtain

**Comment by alexsavio** [comment]
What about this?

**Comment by malemburg** [comment]
This looks fine except for this model change: https://github.com/EuroPython/epcon/pull/388/commits/6f4924570f8b8ac21af439028100ddf68229a23c#diff-9b5efe2c0a0e38c3b0df3f69010e92a4L482

It would need a migration to go with it.

With the migration, the change would be fine.

**Comment by alexsavio** [comment]
I have reverted the VotoTalkAdmin model name change. I don't have time to learn how to prepare a migration file.

**Pull request merged**